utils/api/residents.js

```diff
@@ -12,5 +12,5 @@ export const useResident = (personId) => useSWR(`/api/residents/${personId}`);
 
 export const addResident = async (formData) => {
   const { data } = await axios.post(`/api/residents`, formData);
-  return data;
+  return { ref: data?.personId, data };
 };
```

Comment:
We're not getting the personId back as a ref, resulting in undefined on the create person confirmation page

Reply:
ehm... did you try it?

Reply:
this is actually returning the `personId` as `ref` 👇 
<img width="979" alt="Screenshot 2021-02-26 at 15 01 27" src="https://user-images.githubusercontent.com/435399/109316619-19c32880-784c-11eb-8c33-58a3c4c4a86c.png">